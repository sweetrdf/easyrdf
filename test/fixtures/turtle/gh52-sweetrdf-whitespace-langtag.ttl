@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix : <http://example.org/#> .

# Allow whitespace between literal and language tag
:a rdfs:label "Test"      @en .
