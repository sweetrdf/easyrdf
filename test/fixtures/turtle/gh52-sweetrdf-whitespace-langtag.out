<http://example.org/#a> <http://www.w3.org/2000/01/rdf-schema#label> "Test"@en .
